scripts/redshift/setup_readonly_user.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+"""
+This script creates a readonly user in the redshift db
+"""
+
+from matrix.common.aws.redshift_handler import RedshiftHandler
+from matrix.common.config import MatrixRedshiftConfig
+
+matrix_config = MatrixRedshiftConfig()
+redshift_handler = RedshiftHandler()
+
+
+def handler():
+    readonly_username = matrix_config.readonly_username
+    readonly_password = matrix_config.readonly_password
+    drop_user_query = f"DROP USER IF EXISTS {readonly_username};"
+    add_user_query = f"CREATE USER {readonly_username} WITH PASSWORD '{readonly_password}';"
+    grant_query = f"grant select on all tables in schema public to {readonly_username};"
```

Comment:
I think it needs select on `information_schema` to be able to return available fields dynamically.

Reply:
Well just `information_schema.tables` and `information_schema.columns`

Reply:
okay will grant select on all tables in information_schema

Reply:
will "grant select on all tables in schema information_schema" suffice?

Reply:
Yeah I think that should do it.